src/lib/core/a11y/list-key-manager.ts

```diff
@@ -32,10 +32,18 @@ export class ListKeyManager {
     return this;
   }
 
-  /** Sets the focus of the list to the item at the index specified. */
-  setFocus(index: number): void {
+  /**
+   * Sets the focus of the list to the item at the index specified.
+   *
+   * @param index The index of the item to be focused.
+   * @param focusElement Whether or not to focus the element as well. Defaults to `true`.
+   */
+  setFocus(index: number, focusElement = true): void {
```

Comment:
Calling this in order to perform the focus should still update the index, I was just proposing we add a new method that only updates the index.